test/integration/extensionRequests.test.js

```diff
@@ -214,6 +228,33 @@ describe("Extension Requests", function () {
           return done();
         });
     });
+    it("Should return success response after adding the extension request (sending assignee username)", function (done) {
```

Comment:
i think the message here is misleading.

Reply:
explain further

Reply:
An API always returns a response. a better message tells us what type of response